docker-ee-for-windows/install.md

```diff
@@ -0,0 +1,115 @@
+    ```
+
+2.  Copy the zip file to the machine that isn't connected, by using a thumb drive or by intranet. In a PowerShell command prompt, use the following commands to extract the archive, register, and start the Docker service.
+
+    ```bash
+    #— extract the archive —
+    Expand-Archive docker.zip -DestinationPath $Env:ProgramFiles
+    #clean up zip
+    Remove-Item -Force docker.zip
+    # enable hyper-v if not yet enabled
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All
+
+    #adding docker to PATH
+    $env:path += "$env:ProgramFiles\Docker"
+
+    #register docker daemon as a service
+    dockerd --register-service
+
+    #start the daemon
+    Start-Service docker
+    ```
+
+3.  Test your Docker EE installation by running the `hello-world` container.
+
+    ```
+    docker run hello-world:nanoserver
+    ```
+
+## Install Docker EE by using OneGet
+
+If you want to install Docker EE by using [OneGet](https://github.com/oneget/oneget), follow the steps described in [Windows Containers on Windows Server](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server).
+
+##  What to know before you install
+
+* Nested virtualization scenarios, such as running Docker for Windows
+on a VMWare or Parallels instance, might work, but come with no
+guarantees and are not officially supported. For more info, see
+[Running Docker for Windows in nested virtualization scenarios](/docker-for-windows/troubleshoot.md#running-docker-for-windows-in-nested-virtualization-scenarios).
+
+* **What the Docker EE for Windows install includes**: The installation provides [Docker Engine](/engine/userguide/intro.md), Docker CLI client, [Docker Compose](/compose/overview.md), and [Docker Machine](/machine/overview.md).
+
+### About Docker EE containers and Windows Server 2016
+
+Looking for information on using Docker EE containers?
+
+* [Switch between Windows and Linux containers](/docker-for-windows/index.md#switch-between-windows-and-linux-containers) describes the `Linux/Windows containers` toggle in Docker for Windows.
+
+* [Getting Started with Windows Containers (Lab)](https://github.com/docker/labs/blob/master/windows/windows-containers/README.md)
+provides a tutorial on how to set up and run Windows containers on Windows 10 or Windows Server 2016. It shows you how to use a MusicStore application with
+Windows containers.
+
+* [Setup - Windows Server 2016 (Lab)](https://github.com/docker/labs/blob/master/windows/windows-containers/Setup-Server2016.md) describes environment setup in detail.
+
+* Docker Container Platform for Windows Server 2016 [articles and blog posts](https://www.docker.com/microsoft/) on the Docker website.
+
+## Where to go next
+
+* [Getting started](/docker-for-windows/index.md) provides an overview of Docker for Windows,
```

Comment:
may not be the best place - we don't have docs on using native Windows containers yet.